exercises/all-your-base/canonical-data.json

```diff
@@ -1,23 +1,14 @@
 {
   "exercise": "all-your-base",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "comments": [
-    "It's up to each track do decide:",
+    "This canonical data makes the following choices:",
+    "1. Zero is always represented in outputs as [0] instead of [].",
+    "2. In no other instances are leading zeroes present in any outputs.",
```

Comment:
this is not really a choice that I expect there to be controversy on, so I guess it is just here to be a foil to choice 1.